package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "bitmap-megatron",
+  "version": "1.0.0",
+  "description": "![CF](https://camo.githubusercontent.com/70edab54bba80edb7493cad3135e9606781cbb6b/687474703a2f2f692e696d6775722e636f6d2f377635415363382e706e67) Lab 04: Bitmap Transformer - TEAM MEGATRON === Members: Khalid Mohamud, Dana Kulp, Regan O'Neill",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/reganoneill/04-bitmap_transformer.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/reganoneill/04-bitmap_transformer/issues"
+  },
+  "homepage": "https://github.com/reganoneill/04-bitmap_transformer#readme",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
+    "gulp-mocha": "^3.0.1",
+    "mocha": "^3.2.0",
+    "sinon": "^1.17.7",
+    "sinon-chai": "^2.8.0"
+  }
+}
```

Comment:
@khalidM3 @reganoneill @dkulp23 `package.json` looks as expected - not sure you need `sinon` or `sinon-chai` though (other than maybe for the use of the `spy` method)